docs/package_templates/meson_package/all/conandata.yml

```diff
@@ -2,12 +2,12 @@ sources:
   # Newer versions at the top
   "1.2.0":
     url:
-    - "https://mirror1.net/package-1.2.0.tar.gz",
-    - "https://mirror2.net/package-1.2.0.tar.gz",
+    - "https://mirror1.net/package-1.2.0.tar.gz"
+    - "https://mirror2.net/package-1.2.0.tar.gz"
     sha256: "________________________________________________________________"
   "1.1.0":
-    - "https://mirror1.net/package-1.1.0.tar.gz",
-    - "https://mirror2.net/package-1.1.0.tar.gz",
+    - "https://mirror1.net/package-1.1.0.tar.gz"
+    - "https://mirror2.net/package-1.1.0.tar.gz"
```

Comment:
url is missing

Reply:
Indeed, I missed this extra mistake. Fixed it. Note that this is a template file in the docs. Still, I feel it should show a correct example.